v4/store/nats-js-kv/nats.go

```diff
@@ -335,6 +335,18 @@ func (n *natsStore) String() string {
 	return "NATS JetStream KeyValueStore"
 }
 
+// Watch exposes the watcher interface from the underlying JetStreamContext.
+func (n *natsStore) Watch(bucket string) (nats.KeyWatcher, error) {
```

Comment:
maybe expose WatchAll including options? and as WatchAll? 
```suggestion
func (n *natsStore) WatchAll(bucket string, opts ...nats.WatchOpt) (nats.KeyWatcher, error) {
```

Reply:
Good idea :+1:  Done as requested